command/db-set.go

```diff
@@ -147,3 +160,13 @@ func validateDBSetCommandArgs(projectUUID string, db squarescale.DbConfig) error
 
 	return nil
 }
+
+func checkFlag(dbFlag string, flag *string) string {
+	var result string
+	if *flag == "" {
```

Comment:
:warning: from you function signature flag can be Nil and you don't test it !

Reply:
I set `nil` instead of `""`